doc/modules/preprocessing_targets.rst

```diff
@@ -27,17 +30,38 @@ matrix from a list of multi-class labels::
     array([[1, 0, 0, 0],
            [0, 0, 0, 1]])
 
-For multiple labels per instance, use :class:`MultiLabelBinarizer`::
+For more information about multiclass classification, refer to
+:ref:`multiclass`.
 
-    >>> lb = preprocessing.MultiLabelBinarizer()
-    >>> lb.fit_transform([(1, 2), (3,)])
-    array([[1, 1, 0],
-           [0, 0, 1]])
-    >>> lb.classes_
-    array([1, 2, 3])
+Multilabel classification
```

Comment:
Considering that LabelBinarizer is repurposing the multilabel format in a way that it may not have been intended for, I've included the message using the `. warning::` directive, which I hope makes it clearer to the user that the usage is worth paying attention to.

If it's not too much trouble, could you check the new message in 0d16ce6a2f11d2fca92e96db690a6d2a26d3664a? I think my recommendations are appropriate, but given the topic, I could very well still be misunderstanding. :p